src/core/matrix.c

```diff
@@ -104,6 +118,20 @@ igraph_error_t igraph_matrix_complex_real(const igraph_matrix_complex_t *v,
     return IGRAPH_SUCCESS;
 }
 
+/**
+ * \ingroup matrix
+ * \function igraph_matrix_complex_imag
+ * \brief Gives the imaginary part of a complex matrix.
+ *
+ * \param v Pointer to a complex matrix.
+ * \param imag Pointer to an initialized matrix. The result will be stored here.
+ * \return Error code.
+ *
+ * Time complexity: O(n),
+ * n is the
+ * number of elements in the matrix.
+ */
+
 igraph_error_t igraph_matrix_complex_imag(const igraph_matrix_complex_t *v,
```

Comment:
Let's call the matrix m instead of v, both in the argument list and in the docs.